package.json

```diff
@@ -1,10 +1,10 @@
 {
   "name": "karma-edgium-launcher",
-  "version": "3.1.0",
-  "description": "A Karma plugin. Launcher for Edge and Edge Canary.",
+  "version": "3.1.1",
+  "description": "A Karma plugin. Launcher for Microsoft Edge and Edge Canary.",
   "main": "index.js",
   "scripts": {
-    "lint": "standard",
+    "lint": "eslint .",
```

Comment:
I'd prefer to keep the linting system we already had instead of changing it now.

Reply:
Fair point! I've restored standard as the linter.